src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts

```diff
@@ -91,6 +113,32 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
       this.rgwZoneCount = data['zones'].length;
     });
     this.getPrometheusData(this.prometheusService.lastHourDateObject);
+    this.multisiteSyncStatus$ = this.subject.pipe(
+      switchMap(() =>
+        this.rgwMultisiteService.getSyncStatus().pipe(
+          tap((data: any) => {
+            this.loading = false;
+            this.replicaZonesInfo = data['dataSyncInfo'];
+            this.metadataSyncInfo = data['metadataSyncInfo'];
+            [this.realm, this.zonegroup, this.zone] = data['primaryZoneData'];
+            for (const obj of this.replicaZonesInfo) {
+              if (obj['syncstatus'] === 'Not Syncing From Zone') {
+                obj['syncstatus'] = 'Not Syncing';
+              }
+            }
```

Comment:
why do we go for extra processing inside this and call another for loop. We are already iterating through this object in the html. Can't we just watch for syncstatus === 'Not Syncing from Zone' in the template and just replace it there?